extensions/wikia/Recirculation/WikiRecommendations.class.php

```diff
@@ -610,60 +640,60 @@ class WikiRecommendations {
-			    'title' => 'Midnight Texas',
-			    'url' => 'http://midnight-texas.wikia.com/wiki/Midnight,_Texas_Wikia',
-		    ]
-	    ]
+		'pl' => [
+			[
+				'thumbnailUrl' => 'https://vignette.wikia-dev.pl/gameofthrones/images/3/3a/WhiteWalker_%28Hardhome%29.jpg/revision/latest?cb=20150601151110',
+				'title' => 'Game of Thrones',
+				'url' => 'http://gameofthrones.wikia.com/wiki/Game_of_Thrones_Wiki',
+			],
+			[
+				'thumbnailUrl' => 'https://vignette.wikia-dev.pl/deathnote/images/1/1d/Light_Holding_Death_Note.png/revision/latest?cb=20120525180447',
+				'title' => 'Death Note',
+				'url' => 'http://deathnote.wikia.com/wiki/Main_Page',
+			],
+			[
+				'thumbnailUrl' => 'https://vignette.wikia-dev.pl/midnight-texas/images/b/b0/Blinded_by_the_Light_106-01-Rev-Sheehan-Davy-Deputy.jpg/revision/latest?cb=20170820185915',
+				'title' => 'Midnight Texas',
+				'url' => 'http://midnight-texas.wikia.com/wiki/Midnight,_Texas_Wikia',
+			]
+		]
 	];
 
-	public static function getRecommendations( $contentLanguage ) {
+	public static function getRecommendations($contentLanguage) {
 		global $wgWikiaDatacenter, $wgWikiaEnvironment;
 
-		if ( $wgWikiaEnvironment === WIKIA_ENV_STAGING ) {
+		if ($wgWikiaEnvironment === WIKIA_ENV_STAGING) {
 			$recommendations = self::STAGING_RECOMMENDATIONS;
-		} elseif ( $wgWikiaEnvironment === WIKIA_ENV_DEV ) {
-			if ( $wgWikiaDatacenter === WIKIA_DC_POZ ) {
+		} elseif ($wgWikiaEnvironment === WIKIA_ENV_DEV) {
+			if ($wgWikiaDatacenter === WIKIA_DC_POZ) {
 				$recommendations = self::DEV_RECOMMENDATIONS['pl'];
 			} else {
 				$recommendations = self::DEV_RECOMMENDATIONS['us'];
 			}
 		} else {
 			$recommendations = self::RECOMMENDATIONS['en'];
-			$fallbackedContentLanguage = self::fallbackToSupportedLanguages( $contentLanguage );
+			$fallbackedContentLanguage = self::fallbackToSupportedLanguages($contentLanguage);
 
-			if ( array_key_exists( $fallbackedContentLanguage, self::RECOMMENDATIONS ) ) {
+			if (array_key_exists($fallbackedContentLanguage, self::RECOMMENDATIONS)) {
 				$recommendations = self::RECOMMENDATIONS[$fallbackedContentLanguage];
 			}
-			shuffle( $recommendations );
+			shuffle($recommendations);
 		}
 
-		$recommendations = array_slice( $recommendations, 0, self::WIKI_RECOMMENDATIONS_LIMIT );
+		$recommendations = array_slice($recommendations, 0, self::WIKI_RECOMMENDATIONS_LIMIT);
 
 		$index = 1;
-		foreach($recommendations as &$recommendation) {
-			$recommendation['thumbnailUrl'] = self::getThumbnailUrl( $recommendation['thumbnailUrl'] );
+		foreach ($recommendations as &$recommendation) {
+			$recommendation['thumbnailUrl'] = self::getThumbnailUrl($recommendation['thumbnailUrl']);
 			$recommendation['index'] = $index;
 			$index++;
 		}
 
 		return $recommendations;
 	}
 
-	private static function fallbackToSupportedLanguages( $language ) {
-		switch ( $language ) {
+	private static function fallbackToSupportedLanguages($language) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)